.eslintignore

```diff
@@ -0,0 +1,2 @@
+coverage
+example
```

Comment:
Not sure if you want to lint the example/ directory.
It still has 2 linting errors/warnings, IIRC.